math/clingo/Portfile

```diff
@@ -36,6 +39,11 @@ configure.args-append \
                     -DCLINGO_BUILD_WITH_PYTHON=OFF \
                     -DCLINGO_BUILD_WITH_LUA=OFF
 
+if {${build_arch} eq "ppc" || ${build_arch} eq "ppc64" } {
+    patchfiles-append patch-locale.diff
+    configure.cxxflags-append -latomic
```

Comment:
This is surprising, since `-l` introduces the name of a library to link to, which means this flag is appropriate for the linker only, not for the compiler, so one would expect to find such a flag in LDFLAGS, not in CXXFLAGS.